src/components/header/Breadcrumb.js

```diff
@@ -97,7 +97,10 @@ class Breadcrumb extends Component {
    * @param {*} menu
    */
   handleClick = (e, menu) => {
-    const { handleMenuOverlay, windowType } = this.props;
+    const { handleMenuOverlay, windowType, filters, dispatch } = this.props;
+    if (!filters.clearAll) {
+      dispatch(clearAllFilters(true));
```

Comment:
maybe instead of passing `dispatch` just throw `clearAllFilters` in the connect, like we have elsewhere in the app ?

Reply:
You mean add it within mapDispatchToProps.  You see in this file everything is called with dispatch so I would leave this for the next refactoring as this is an urgent fix that needs to reach master.  Basically every action would need to reside in mapDispatchToProps. Thanks for the suggestion